orangecontrib/associate/widgets/owassociate.py

```diff
@@ -8,31 +8,37 @@
 from Orange.data import Table
 from Orange.widgets import widget, gui, settings
 from Orange.widgets.visualize.owscatterplotgraph import OWScatterPlotGraph
+from Orange.widgets.widget import Input, Output
 
-from PyQt4.QtCore import Qt, QSize, pyqtSignal, QRectF
-from PyQt4.QtGui import (
-    QApplication, QLabel, QTableView, QStandardItem, QStandardItemModel,
-    QSortFilterProxyModel, QMainWindow, QMouseEvent, QGraphicsView,
-    QPen, QBrush, QColor)
-
+from AnyQt.QtCore import Qt, QSize, pyqtSignal, QRectF, QSortFilterProxyModel
+from AnyQt.QtGui import (
+    QApplication, QStandardItem, QStandardItemModel, QMouseEvent, QPen, QBrush, QColor)
+from AnyQt.QtWidgets import QLabel, QTableView, QMainWindow, QGraphicsView
 
 from orangecontrib.associate.fpgrowth import frequent_itemsets, OneHot, association_rules, rules_stats
 
 import pyqtgraph as pg
 
 
-class Output:
-    DATA = 'Matching data'
-
-
 class OWAssociate(widget.OWWidget):
     name = 'Association Rules'
     description = 'Induce association rules from data.'
     icon = 'icons/AssociationRules.svg'
     priority = 20
 
-    inputs = [("Data", Table, 'set_data')]
-    outputs = [(Output.DATA, Table)]
+    class Inputs:
+        data = Input("Data", Table)
+
+    class Outputs:
+        matching_data = Output("Matching data", Table)
```

Comment:
Sorry, I meant the labels. We title case them.

Reply:
Ok.